packages/flutter_tools/lib/src/tracing.dart

```diff
@@ -72,11 +73,25 @@ class Tracing {
           }
         }
         if (!done) {
-          final Timer timer = Timer(const Duration(seconds: 10), () {
+          final Timer timer = Timer(const Duration(seconds: 10), () async {
             _logger.printStatus('First frame is taking longer than expected...');
-            _logger.printTrace('Views:');
             for (final FlutterView view in views) {
-              _logger.printTrace('id: ${view.id} isolate: ${view.uiIsolate?.id}');
+              final String? isolateId = view.uiIsolate?.id;
+              _logger.printTrace('View ID: ${view.id}');
+              if (isolateId == null) {
+                _logger.printTrace('No isolate ID associated with the view.');
+                continue;
+              }
+              final vm_service.Isolate? isolate = await vmService.getIsolateOrNull(isolateId);
```

Comment:
We have the isolate here, is there anything else helpful to add?

Reply:
How about `getStack()` https://github.com/dart-lang/sdk/blob/main/runtime/vm/service/service.md#getstack ? Not sure what the Dart API for that is, though.

Reply:
That doesn't work in profile mode.
```
RPCError: getStack: (100) Feature is disabled
Debugger is disabled in AOT mode.

```
#0      new _OutstandingRequest (package:vm_service/src/vm_service.dart:1734:45)
#1      VmService._call (package:vm_service/src/vm_service.dart:2245:21)
#2      VmService.getStack (package:vm_service/src/vm_service.dart:2021:61)
#3      Tracing.stopTracingAndDownloadTimeline.<anonymous closure> (package:flutter_tools/src/tracing.dart:85:70)
```